cj10-screaming-snakecases/Engine/ImageViewer.py

```diff
@@ -0,0 +1,9 @@
+# This file contains the class ImageViewer, which should show pixel-level statistics of a certain region of interest
+# For example: Show histograms, color channels, and pixel-level details. (@Envxsion)
+
+# This will have to be programmed in conjunction with the GUI soon.
+
+
+class ImageViewer():
+    def __init__(self) -> None:
+        pass
```

Comment:
```suggestion
        pass

```
There should be 1 newline at the end of the file